Language/Functions/Communication/serial.adoc

```diff
@@ -15,17 +15,17 @@ subCategories: [ "Communication" ]
-[%hardbreaks]
-Serial communication on pins TX/RX uses TTL logic levels (5V or 3.3V depending on the board). Don't connect these pins directly to an RS232 serial port; they operate at +/- 12V and can damage your Arduino board.
-[%hardbreaks]
-The *Arduino Mega* has three additional serial ports: `Serial1` on pins 19 (RX) and 18 (TX), `Serial2` on pins 17 (RX) and 16 (TX), `Serial3` on pins 15 (RX) and 14 (TX). To use these pins to communicate with your personal computer, you will need an additional USB-to-serial adaptor, as they are not connected to the Mega's USB-to-serial adaptor. To use them to communicate with an external TTL serial device, connect the TX pin to your device's RX pin, the RX to your device's TX pin, and the ground of your Mega to your device's ground.
-[%hardbreaks]
-The *Arduino DUE* has three additional 3.3V TTL serial ports: `Serial1` on pins 19 (RX) and 18 (TX); `Serial2` on pins 17 (RX) and 16 (TX), `Serial3` on pins 15 (RX) and 14 (TX). Pins 0 and 1 are also connected to the corresponding pins of the ATmega16U2 USB-to-TTL Serial chip, which is connected to the USB debug port. Additionally, there is a native USB-serial port on the SAM3X chip, SerialUSB'.
-[%hardbreaks]
-The *Arduino Leonardo* board uses `Serial1` to communicate via TTL (5V) serial on pins 0 (RX) and 1 (TX). `Serial` is reserved for USB CDC communication. For more information, refer to the Leonardo getting started page and hardware page.
+=== Descrizione
+Utilizzato per la comunicazione tra la scheda Arduino e un computer o altri dispositivi. Tutte le schede Arduino hanno almeno una porta seriale (nota anche come UART o USART): Seriale. Comunica sui pin digitali 0 (RX) e 1 (TX) oltre che con il computer tramite USB. Pertanto, se si utilizzano queste funzioni, non è possibile utilizzare anche i pin 0 e 1 per l'ingresso o l'uscita digitale.
+È possibile utilizzare il monitor seriale integrato nell'ambiente Arduino per comunicare con una scheda Arduino. Fare clic sul pulsante del monitor seriale nella barra degli strumenti e selezionare la stessa velocità di trasmissione utilizzata nella chiamata a `begin ()`.
+[% hardbreaks]
+La comunicazione seriale sui pin TX / RX utilizza livelli logici TTL (5V o 3.3V a seconda della scheda). Non collegare questi pin direttamente a una porta seriale RS232; funzionano a +/- 12V e possono danneggiare la tua scheda Arduino.
+[% hardbreaks]
+* Arduino Mega * ha tre porte seriali aggiuntive: `Serial1` sui pin 19 (RX) e 18 (TX),` Serial2` sui pin 17 (RX) e 16 (TX), `Serial3` sui pin 15 (RX) e 14 (TX). Per utilizzare questi pin per comunicare con il tuo personal computer, avrai bisogno di un ulteriore adattatore da USB a seriale, poiché non sono collegati all'adattatore da USB a seriale di Mega. Per usarli per comunicare con un dispositivo seriale TTL esterno, collega il pin TX al pin RX del tuo dispositivo, l'RX al pin TX del tuo dispositivo e la terra del tuo Mega alla terra del tuo dispositivo.
```

Comment:
```suggestion
*Arduino Mega* ha tre porte seriali aggiuntive: `Serial1` sui pin 19 (RX) e 18 (TX),` Serial2` sui pin 17 (RX) e 16 (TX), `Serial3` sui pin 15 (RX) e 14 (TX). Per utilizzare questi pin per comunicare con il tuo personal computer, avrai bisogno di un ulteriore adattatore da USB a seriale, poiché non sono collegati all'adattatore da USB a seriale di Mega. Per usarli per comunicare con un dispositivo seriale TTL esterno, collega il pin TX al pin RX del tuo dispositivo, l'RX al pin TX del tuo dispositivo e la terra del tuo Mega alla terra del tuo dispositivo.
```
Leading or trailing spaces in the `*` cause the contents to not receive the intended bold formatting.